tests/Microsoft.Identity.Web.Test.Integration/Microsoft.Identity.Web.Test.Integration.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <SignAssembly>true</SignAssembly>
```

Comment:
Do we need to sign test assemblies?

Reply:
As far as I know you need to sign all assemblies (or I should call it "strongly name"). AFAIK you cannot mix strongly-named and non-strongly-named assemblies. I may be wrong, there might be some subtleties around this. In any case, this is what is done in MSAL.

Reply:
Makes sense. Thanks. 👍